### PR TITLE
[Fix docs] Add Azure supported images

### PR DESCRIPTION
**Pull request by plirglo**

### CODE REVIEW

File: docs/home/howto/CLUSTER.md [hunk 1]
```diff
@@ -292,6 +292,35 @@ To setup the cluster do the following steps from the provisioning machine:
...
+Depending on your requirements you may need to create cluster based on Red Hat family (RHEL/CentOS) system. Be aware that Epiphany supports only images with RAW partitioning, and attached to standard RHEL repositories. Please read this article for more details:
+
+https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-images#rhel-7-image-types
+
+It means that actually for Azure Cloud Epiphany supports Red Hat 7.6 - 7.7 and CentOS 7.6 - 7.8 Versions (with RAW partitioning and attached to RHEL repositories).
+List of supported images will be extended in next releases.
+
+Example config for Red Hat:
+
+```
```
Comment (seriva) [inline docs/home/howto/CLUSTER.md:306]: Use:
'''yaml

File: docs/home/howto/CLUSTER.md [hunk 2]
```diff
@@ -292,6 +292,35 @@ To setup the cluster do the following steps from the provisioning machine:
...
+specification:
+  storage_image_reference:
+    publisher: RedHat
+    offer: RHEL
+    sku: 7-RAW
+    version: "7.7.2019090418"
+```
+Example config for CentOS:
+```
```
Comment (seriva) [inline docs/home/howto/CLUSTER.md:315]: Use:
'''yaml

File: docs/home/howto/CLUSTER.md [hunk 3]
```diff
@@ -292,6 +292,35 @@ To setup the cluster do the following steps from the provisioning machine:
     epicli apply -f newcluster.yml
     ```
 
+## Note for Red Hat / CentOS Azure images
```
Comment (seriva) [inline docs/home/howto/CLUSTER.md:295]: 3 times # if this is a subpart of "How to create an Epiphany cluster on a cloud provider"

**Review by seriva — CHANGES_REQUESTED** [review]

File: docs/home/howto/CLUSTER.md (hunk 1 above)
Comment (seriva) [reply docs/home/howto/CLUSTER.md:306]: Also maybe make sure that its know that these changes need to be made in the infrastructure/virtual-machine documents.

**Review by seriva — CHANGES_REQUESTED** [review]

File: docs/home/howto/CLUSTER.md (hunk 1 above)
Comment (plirglo) [reply docs/home/howto/CLUSTER.md]: Added

File: docs/home/howto/CLUSTER.md (hunk 2 above)
Comment (plirglo) [reply docs/home/howto/CLUSTER.md]: Added yaml and "kind" part

File: docs/home/howto/CLUSTER.md (hunk 3 above)
Comment (plirglo) [reply docs/home/howto/CLUSTER.md]: Added subpart

**Review by seriva — APPROVED** [review]

File: docs/home/howto/CLUSTER.md [hunk 4]
```diff
@@ -292,6 +292,37 @@ To setup the cluster do the following steps from the provisioning machine:
     epicli apply -f newcluster.yml
     ```
 
+### Note for Red Hat / CentOS Azure images
+
+Depending on your requirements you may need to create cluster based on Red Hat family (RHEL/CentOS) system. Be aware that Epiphany supports only images with RAW partitioning, and attached to standard RHEL repositories. Please read this article for more details:
+
+https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-images#rhel-7-image-types
```
Comment (to-bar) [inline docs/home/howto/CLUSTER.md:299]:
```suggestion
For RHEL and CentOS, Epiphany currently supports only images with RAW partitioning and attached to standard RHEL repositories. For more details, refer to [Azure documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-images#rhel-7-image-types).
```

File: docs/home/howto/CLUSTER.md [hunk 5]
```diff
@@ -292,6 +292,37 @@ To setup the cluster do the following steps from the provisioning machine:
...
+
+It means that actually for Azure Cloud Epiphany supports Red Hat 7.6 - 7.7 and CentOS 7.6 - 7.8 Versions (with RAW partitioning and attached to RHEL repositories).
```
Comment (to-bar) [inline docs/home/howto/CLUSTER.md:301]:
```suggestion
It means that actually for Azure Cloud Epiphany supports RHEL 7.6 - 7.7 and CentOS 7.6 - 7.8 versions (with RAW partitioning and attached to standard repositories).
```

File: docs/home/howto/CLUSTER.md [hunk 6]
```diff
@@ -292,6 +292,37 @@ To setup the cluster do the following steps from the provisioning machine:
...
+List of supported images will be extended in next releases.
+
+Example config for Red Hat:
```
Comment (to-bar) [inline docs/home/howto/CLUSTER.md:304]:
```suggestion
Example config for RHEL:
```

**Review by to-bar — CHANGES_REQUESTED** [review]

**Review by to-bar — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Pull request merged**